Formula/asdf.rb

```diff
@@ -4,6 +4,7 @@ class Asdf < Formula
   url "https://github.com/asdf-vm/asdf/archive/v0.8.1.tar.gz"
   sha256 "6ca280287dcb687ec12f0c37e4e193de390cdab68f2b2a0e271e3a4f1e20bd2e"
   license "MIT"
+  revision 1
```

Comment:
Wouldn't this really be revision 2?

Reply:
An omitted `revision` is implicitly `revision 0`.